app/src/androidTest/java/com/github/se/signify/ui/screens/profile/ProfileScreenTest.kt

```diff
@@ -126,20 +126,7 @@ class ProfileScreenTest {
   fun dialogIsDisplayedWhenHelpBoxIsVisible() {
 
     // Assert that the help dialog is displayed when the help button is clicked
-    composeTestRule.onNodeWithTag("InfoButton").performClick()
-    composeTestRule.onNodeWithTag("InfoPopupContent").assertIsDisplayed()
-  }
-
-  @Test
-  fun dialogClosesWhenCloseButtonClicked() {
```

Comment:
Is there a specific reason why this test was deleted? I’m sure it wouldn’t affect the overall coverage significantly, but I’m curious.

Reply:
I've deleted the close button entirely. The test can't exist anymore.

Reply:
Also there is no reason for the profile screen to be testing common elements anyway. I've let tests that do this be for now, when they didn't break. But ideally, only `Screens.kt` should be testing its elements, for modularity.